pkgs/applications/graphics/zgv/png.patch

```diff
@@ -0,0 +1,47 @@
+diff -ur zgv-5.9/config.mk zgv-5.9.new/config.mk
```

Comment:
@joachifm Does this help? I have clarified the patch (due to interface change from libpng 1.2 -> 1.4)